### DISCUSSION TITLE
Add Installation with cuda to documentation

### DISCUSSION THREAD

**Discussion by oscarbau** [opengeos/segment-geospatial#185] [Ideas]
Hi @giswqs,

I realized that the default installation procedure described in the documentation installs the pytorch-only CPU version. I suggest you include a section for when you want to install segment-geospatial including pytorch with GPU as a default in the installation tutorial.

The steps that I have followed are:
creation of the fresh environment
install cuda toolkit
install a pytorch GPU version that matches the cuda toolkit version
and then install segment geospatial.

Hope these steps are the correct ones and that the idea is useful to improve the documentation.

**Comment by giswqs** [comment]
If your system has GPU and drivers have been installed properly, the mamba commands will install the gpu version properly. Check out the links below on how to set up gpu properly on Windows. 

- http://www.spatialpoints.com/2022/04/20/deep-learning-in-arcgis-pro-using-gpu/
- https://www.esri.com/arcgis-blog/products/arcgis-pro/imagery/deep-learning-with-arcgis-pro-tips-tricks/
